### PR TITLE
Updated Joining Testnet Tutorial

### PR DESCRIPTION
**Pull request by dasanchez**
Updated the testnet tutorial with the current `v9.0.0-rc3` version.

### CODE REVIEW

File: docs/hub-tutorials/join-testnet.md [hunk 1]
```diff
@@ -62,14 +63,14 @@ Install build tools and Go.
...
-sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.19.4.linux-amd64.tar.gz
+wget https://go.dev/dl/go1.18.5.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.18.5.linux-amd64.tar.gz
 export PATH=$PATH:/usr/local/go/bin
 ```
 
 ### Installation & Configuration
 
-You will need to install and configure the Gaia binary using the script below. The Cosmos Hub Public Testnet is running Gaia [`v8.0.0-rc3`](https://github.com/cosmos/gaia/releases/tag/v8.0.0-rc3).
+You will need to install and configure the Gaia binary using the script below. The Cosmos Hub Public Testnet is running Gaia [`v9.0.0-rc3`](https://github.com/cosmos/gaia/releases/tag/v9.0.0-rc3).
```
Comment (mmulji-ic) [inline docs/hub-tutorials/join-testnet.md]: I guess when out of draft: `v9.0.0-rc3` -> `v9.0.0-rc4`

**Review by mmulji-ic — APPROVED** [review]

**Review by yaruwangway — APPROVED** [review]

File: docs/hub-tutorials/join-testnet.md [hunk 2]
```diff
@@ -79,7 +80,7 @@ cd $HOME
 git clone https://github.com/cosmos/gaia
 cd gaia
 # To sync from genesis, comment out the next line.
-git checkout v8.0.0-rc3
+git checkout v9.0.0-rc3
```
Comment (yaruwangway) [inline docs/hub-tutorials/join-testnet.md]:
```suggestion
git checkout v9.0.0-rc6
```

File: docs/hub-tutorials/join-testnet.md (hunk 1 above)
Comment (yaruwangway) [inline docs/hub-tutorials/join-testnet.md]:
```suggestion
You will need to install and configure the Gaia binary using the script below. The Cosmos Hub Public Testnet is running Gaia [`v9.0.0-rc6`](https://github.com/cosmos/gaia/releases/tag/v9.0.0-rc6).
```

**Review by yaruwangway — APPROVED** [review]

**Comment by yaruwangway** [comment]
@dasanchez , can you update the branch of v9 version, we can merge after that. Thank you!

**Pull request merged**